Cargo.toml

```diff
@@ -63,6 +61,10 @@ futures-util = "0.3.30"
 nispor = "1.2.19"
 tower = { version = "0.4.13" }
 
+[target.'cfg(not(target_os = "freebsd"))'.dependencies]
```

Comment:
iptables and nftables are Linux-kernel-specific, should this be == Linux rather than != FreeBSD?

Reply:
Good point. My rationale here, and in other places, is that netavark supports Linux by default so I expressed the intentions from freebsd, as in whether or not target platform is freebsd, perspective.

What do you think ?

Reply:
We need the project's perspective I think -- checking that it *is* Linux will be more portable in the future if someone is looking to add support for yet another OS and IMO it's easier to reason about this in the positive sense.

Reply:
My personal opionion is that checking for ==linux is more correct but I completely agree that this is a question for the project owners.